src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/GatewayOverviewOrchestratorTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AdminService.Web.Infrastructure.RoatpClients;
+using SFA.DAS.AdminService.Web.Services.Gateway;
+using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+using SFA.DAS.AssessorService.ApplyTypes.Roatp.Apply;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AdminService.Web.Tests.Services.Gateway
+{
+    [TestFixture]
+    public class GatewayOverviewOrchestratorTests
+    {
+        private GatewayOverviewOrchestrator _orchestrator;
+        private Mock<IRoatpApplicationApiClient> _applyApiClient;
+        private Mock<ILogger<GatewayOverviewOrchestrator>> _logger;
+        private Mock<IGatewaySectionsNotRequiredService> _sectionsNotRequiredService;
+
+        private static string UserName = "GatewayUser";
+
+        [SetUp]
+        public void Setup()
+        {
+            _applyApiClient = new Mock<IRoatpApplicationApiClient>();
+            _logger = new Mock<ILogger<GatewayOverviewOrchestrator>>();
+            _sectionsNotRequiredService = new Mock<IGatewaySectionsNotRequiredService>();
+            _orchestrator = new GatewayOverviewOrchestrator(_applyApiClient.Object, _logger.Object, _sectionsNotRequiredService.Object);
+        }
+
+        [TestCase("12345678", "John Ltd.", SectionReviewStatus.Pass, "Very good.")]
+        [TestCase("87654321", "Simon Ltd.", SectionReviewStatus.Fail, "Not so good.")]
+        [TestCase("12344321", "Frank Ltd.", SectionReviewStatus.NotRequired, null)]
+        public void Check_GetConfirmOverviewViewModel_returns_model(string ukprn, string organisationName, string sectionReviewStatus, string comment)
```

Comment:
There are a number of if statements in the code being tested but only one test which doesn't seem right.

Reply:
Which one and why? So, I can address the issue.

Reply:
All of them because all paths through the code should be tested?  This is why we should be following TDD, so we don't end up with untested code.

Reply:
More coverage was added.